src/main/java/Controller/userController.java

```diff
@@ -1,4 +1,37 @@
 package Controller;
 
+import InputBoundary.loginInputBoundary;
+import InputData.loginInputData;
+
+
+/**
+ * Controller for login and register use cases.
+ */
 public class userController {
+    final loginInputBoundary loginInteractor, registerInteractor;
```

Comment:
Shouldn't these be private? I think if you don't specify then it will be made public by the compiler but it should be private. Could you please add the private keyword? Can you also please make these on seperate lines please?